tests/ci/cdk/cdk/codebuild/github_ci_integration_omnibus.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
+batch:
+  build-list:
+    - identifier: s2n_integration
+      buildspec: ./tests/ci/codebuild/integration/s2n_integration.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_SMALL
+        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-x86:ubuntu-20.04_clang-9x_latest
+
+    - identifier: openssh_integration
+      buildspec: ./tests/ci/codebuild/integration/openssh_integration.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_SMALL
```

Comment:
Have you deployed and run this change in your account? Looking at our existing CI runs OpenSSH takes 58 minutes to run on a large and a small could take 4 hours (assuming perfect scaling). However, if all that time is spent in a single thread test a small would be fine.

Reply:
Good point, I applied the CI change to my personal account and it seems like the test dimension is single threaded. It still takes 55 minutes with a SMALL instance on my account.
* https://github.com/samuel40791765/aws-lc/pull/24